README.md

```diff
@@ -1,84 +1,57 @@
-<img src='https://s3-us-west-2.amazonaws.com/cortex-public/logo.png' height='42'>
-
-<br>
-
 [Website](https://www.cortex.dev) • [Slack](https://community.cortex.dev) • [Docs](https://docs.cortex.dev)
```

Comment:
Maybe copy this (or a different call to action, e.g. "install") at the bottom of the file?